### DISCUSSION TITLE
Tracking overlapping teams schedules

### DISCUSSION THREAD

**Discussion by boggsdabeast** [vasqued2/ha-teamtracker#98] [Ideas]
Hello There!

Love the integration with the card for dashboards!

This may not be the right place to post this, but since since this is so awesome, figured I'd throw this out.
Is there a way (this/new/repo) integration could tell a user a date range for sports teams that I follow; when any of them are playing away games within a given timespan?
The idea would be my basketball and hockey team, or baseball team and my soccer team are away in the same city a the same time (or even baseball and baseball with away back to back series in same state/region/province) or any overlapping combinations; so I could try and plan a vacation around that timeframe to get a two/three/four/etc sporting event for one vacation.

Hope this question make sense.

**Comment by vasqued2** [comment]
There's no way to do it with this integration.

Here is a link with a whole bunch of ESPN APIs.  It looks like there is one that brings back a teams schedule for the year.  You might be able to pull multiple schedules back with it and then look for overlaps based off the venue address.

https://gist.github.com/nntrn/ee26cb2a0716de0947a0a4e9a157bc1c
